src/sentry/analytics/event.py

```diff
@@ -9,23 +9,43 @@
 
 
 class Attribute(object):
-    __slots__ = ['name', 'type', 'required']
-
     def __init__(self, name, type=six.text_type, required=True):
         self.name = name
         self.type = type
         self.required = required
 
+    def serialize(self, value):
```

Comment:
what would you call it?

(i would argue it is serialization, as its serializing to a specified format)